### PR TITLE
Header

### PR DESCRIPTION
**Pull request by chenrygit**

### CODE REVIEW

File: new-components/header.html [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+      width: 960px;
+      margin: 0 auto;
+    }
+    
+  </style>
+
+  <style>
+
+    /* Primary Menu "Main Menu" */
+    .menu {
```
Comment (orderedlist) [inline new-components/header.html]: For consistency with other components, should you preface these styles with `.v2`?

File: new-components/header.html [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+      margin: 0;
+      display: table-cell;
+      vertical-align: middle;
+      color: #fff;
+      font-weight: 600;
+      text-transform: uppercase;
+      text-decoration: none;
+      min-width:2em;
+    }
+    a.menu-link.active, a.menu-link.active:hover {
```
Comment (orderedlist) [inline new-components/header.html]: Are the `a` selectors here necessary for overriding the cascade? Can they be removed if not?

File: new-components/header.html [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+    }
+
+    /* Application context label */
+    .menu-link.menu-app-label {
+      padding: 10px 25px;
+      font-size:24px;
+      font-weight:300;
+      cursor:default;
+    }
+    a.menu-link.menu-app-label:hover {
```
Comment (orderedlist) [inline new-components/header.html]: Same here with the `a`

File: new-components/header.html [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+      top: 100%;
+      left: -1px;
+      background: #fff;
+      border: 1px solid #e6e7e8;
+      margin-top: -1px;
+      z-index: 100;
+    }
+    .menu-dropdown .menu-dropdown-links {
+      width: 240px;
+      margin-left:-41px;
```
Comment (orderedlist) [inline new-components/header.html]: This px seems really specific. What's it trying to accomplish?

File: new-components/header.html [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+    .sub-menu {
+      display:block;
+      background:#fff;
+    }
+    .sub-menu-links {
+      display: table-cell;
+      margin: 0;
+      vertical-align: middle;
+      width:100%;
+      width:100%;
```
Comment (orderedlist) [inline new-components/header.html]: Duplicated `width`.

File: new-components/header.html [hunk 6]
```diff
@@ -0,0 +1,512 @@
...
+    }
+    a.sub-menu-link {
```
Comment (orderedlist) [inline new-components/header.html]: Same comment here with the `a`

File: new-components/header.html [hunk 7]
```diff
@@ -0,0 +1,512 @@
...
+      margin: 0;
+      vertical-align: middle;
+      font-size:16px;
+      font-weight: 300;
+      line-height: 1.4;
+      color: #0b1f66;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+    a.sub-menu-link:hover {
```
Comment (orderedlist) [inline new-components/header.html]: `a`?

File: new-components/header.html [hunk 8]
```diff
@@ -0,0 +1,512 @@
...
+      color: #0b1f66;
+      background-color: #ebf0f4;
+    }
+    a.sub-menu-link.active {
```
Comment (orderedlist) [inline new-components/header.html]: `a`?

File: new-components/header.html [hunk 9]
```diff
@@ -0,0 +1,512 @@
...
+    .menu-login-title {
+      font-weight: 600;
+      font-size: 14px;
+      padding: 0 24px;
+      display: block;
+      text-transform: uppercase;
+    }
+
+    /* Single-use toggleable fixed-width elements */
+    #menu-tab-admin{
```
Comment (orderedlist) [inline new-components/header.html]: Could these be made into more generic classes to avoid specific naming? `.menu-tab-sm`, `.menu-tab-md`, `.menu-tab-lg` for example?

File: new-components/header.html [hunk 10]
```diff
@@ -0,0 +1,512 @@
...
+
+   Notes:
+    - Never use inline styles.
+    - During integration, ignore DIVs with classes that have 'testing' in their name.
+   ///////////////////////////////////////////-->
+
+  <body class="v2">
+
+
+    <div class="testing-header testing-header-only-links">
```
Comment (orderedlist) [inline new-components/header.html]: Perhaps including an html comment after these next two divs to specify they are not part of the component?

File: new-components/header.html (hunk 1 above)
Comment (chenrygit) [inline new-components/header.html]: Since the new header will be replacing the v1 header (also used on v2 sites), I didn't utilize the v2 namespace. We're essentially stabilizing an existing v1 component.

File: new-components/header.html (hunk 2 above)
Comment (chenrygit) [inline new-components/header.html]: For some reason it wasn't working without the 'a' specified.  I'll test again and focus some more effort towards identifying why that was/is happening.

File: new-components/header.html (hunk 5 above)
Comment (chenrygit) [inline new-components/header.html]: fixed

File: new-components/header.html (hunk 1 above)
Comment (chenrygit) [inline new-components/header.html]: With that said I'm removing the <body class="v2">, so the context is more clear.

File: new-components/header.html (hunk 4 above)
Comment (chenrygit) [inline new-components/header.html]: This is used to align the dropdown with the right side of it's parent menu item.   The extra pixel is there to offset a 1px border.

File: new-components/header.html (hunk 6 above)
Comment (chenrygit) [inline new-components/header.html]: fixed

File: new-components/header.html (hunk 7 above)
Comment (chenrygit) [inline new-components/header.html]: fixed

File: new-components/header.html (hunk 8 above)
Comment (chenrygit) [inline new-components/header.html]: fixed

File: new-components/header.html (hunk 3 above)
Comment (chenrygit) [inline new-components/header.html]: fixed

File: new-components/header.html (hunk 2 above)
Comment (chenrygit) [inline new-components/header.html]: I was able to remove the anchor 'a', wasn't a problem.  However we need to continue concatenating paired class names for required specificity.

File: new-components/header.html (hunk 9 above)
Comment (chenrygit) [inline new-components/header.html]: I chose to use IDs because these styles are specific to the use of tab being targeted.  Since they are so specific and there's no need to reuse this specific of a width on other elements I chose not to make them IDs instead of classes.   It's also easier to quickly differentiate if anything needs to be tweaked in the future.

File: new-components/header.html (hunk 10 above)
Comment (chenrygit) [inline new-components/header.html]: I've included this in the comments for 'integration guidelines'.

It says: 
"During integration, ignore DIVs with classes that have 'testing' in their name."

**Comment by chenrygit** [comment]
Does anyone else have any more code review feedback for this component?

File: new-components/performance-statistics-competitor.html [hunk 11]
```diff
@@ -0,0 +1,139 @@
+<head>
```
Comment (orderedlist) [inline new-components/performance-statistics-competitor.html]: Is this file intended to be in this PR?

File: new-components/company-name-agency.html [hunk 12]
```diff
@@ -13,9 +13,9 @@
     color: #ffffff;
     font-weight: 300;
     text-align: left;
-    margin-left: auto;
```
Comment (orderedlist) [inline new-components/company-name-agency.html]: Are these edits intended to be in this PR?
